jobs.groovy.j2

```diff
@@ -90,6 +90,28 @@
 {# JINJA2 MACROS DEFINED BELOW                                               #}
 {# --------------------------------------------------------------------------#}
 
+{% macro updateGitHubStatus(status, description, branch, dashProject,
+                            dashBranchName)                                 -%}
+"""
+#!/bin/bash
+set +x
```

Comment:
maybe state why we force a +x

# Jenkins runs jobs using 'set -x'
# we set '+x' so that we don't leak credentials to the logs
set +x